ui/src/views/Documentation/Reference/ApiReference.jsx

```diff
@@ -49,13 +69,20 @@ export default class ApiReference extends Component {
               the manual.
             </Typography>
             <br />
-            {functionEntries.map(entry => (
-              <Entry
-                key={`${entry.name}-${entry.query}`}
-                type="function"
-                entry={entry}
-                serviceName={ref.serviceName}
-              />
+            {groupedEntries.map(group => (
+              <div key={group[0]}>
```

Comment:
Use a `Fragment` instead of a `div`. This will make sure the following style is applied:

https://github.com/taskcluster/taskcluster/blob/dc700ea3b0690967fcf0af585d9b34cf60307a7d/ui/src/views/Documentation/components/HeaderWithAnchor/index.jsx#L12-L14

Reply:
Applied the Changes!